_posts/2021-08-23-rocket-validator.md

```diff
@@ -0,0 +1,119 @@
+
+Low vision may make a web unreadable if the text has low contrast or the font size cannot be adapted easily. Unsighted people can use screen readers to hear the page content, but it won't work well if the content is poorly structured and can't be properly interpreted by the screen reader software. People with reduced mobility may prefer to navigate through the lessons using the keyboard instead of the mouse, but again this won't work fine if the document is not laid out correctly.
+
+The good news is that we can make Elixir School accessible for everyone. And it's not a hard task, it just requires that we're aware of those barriers and that we follow the web standards to provide well-structured web pages.
+
+## How to find accessibility and HTML issues
+
+The World Wide Web Consortium (W3C) is an international community that defines and develops [Web standards](https://www.w3.org/standards/), HTML being probably the most used -and abused- of those standards. At the end of 1997, the [W3C launched](https://www.webdesignmuseum.org/web-design-history/w3c-html-validator-1997) the [W3C HTML Validator](https://validator.w3.org/), a service to check any web page and find issues in its HTML structure. More than two decades later this service is still around and in good health and many tools have been built around its [open source validator](https://github.com/validator/validator).
+
+HTML is the basis of web pages, but still a perfectly valid HTML page can have accessibility issues. For example, a web page may have a bad combination of colors that results in low contrast: a light color text over a light background may lead to unreadable text by people with low vision. If a video has no captions, deaf users won't understand what it's talking about. Sections of the document may be not accessible for users that don't employ a mouse to move through the document. Zooming and scaling may have been disabled, making it impossible for the users to enlarge the web page to see it better. Many known issues are easily detected using automated accessibility checkers like [axe-core](https://github.com/dequelabs/axe-core).
+
+## The power of automated site validation
+
+So, it's great that we have tools that help in detecting issues on web pages. Both the W3C HTML validator and axe-core are open source and can be incorporated into our web development flow. But, there's an important limitation: they can only check one URL at a time. When you have a small website with just a few web pages, it might be an option to do manual checks using these tools. But, what can we do with a large site like Elixir School? On the latest reports, Elixir School has more than 1,000 web pages. If we want to check them all with the W3C Validator and axe-core, we're going to run 2,000 manual checks!
```

Comment:
```suggestion
So, it's great that we have tools that help in detecting issues on web pages. Both the W3C HTML validator and axe-core are open source and can be incorporated into our web development flow. But, there's an important limitation: they can only check one URL at a time. When you have a small website with just a few web pages, it might be an option to do manual checks using these tools. But, what can we do with a large site like Elixir School? According to the latest reports, Elixir School has more than 1,000 web pages. If we want to check them all with the W3C Validator and axe-core, we're going to run 2,000 manual checks!
```